buildspec.yml

```diff
@@ -0,0 +1,69 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - apk update
+      - apk add python3
+      - apk add python3-dev
```

Comment:
I understand that these are here b/c you're using the `hashicorp/terraform` image but this build env might be easier to maintain in the long run if we stick with the AL2 image and just install terraform on build. We have a make target in tardigrade-ci that handles the terraform install that you can steal to make things that much easier

Reply:
My longer term goal was to create a container that targeted the specific versions of software used for the project and skip the installs altogether.